extras/installer.sh

```diff
@@ -277,6 +277,9 @@ fi
 
 echo
 read -e -p "Directory to install into: " -i "/opt/plexupdate" FULL_PATH
+if [[ "$FULL_PATH" == *"~"* ]]; then # If path contains a tilde
```

Comment:
In that case, then if we detect a ~ in the path, should we just do something like:

    if [[ "$FULL_PATH" == *"~"* ]]; then
        abort "Invalid character in path"
    fi

And if that is the case, are there any other characters we should be trapping?